librato/queue.py

```diff
@@ -72,7 +71,7 @@ def add_aggregator(self, aggregator):
             nm['name'] = name
             # Set measure_time
             if mt:
-              nm['measure_time'] = mt
+                nm['measure_time'] = mt
```

Comment:
Never mind, I guess you're going for pep8 or whatever the cool kids use in Python these days.